aws/hhvm1/lambdas/test.py

```diff
@@ -143,12 +143,12 @@ def test_get_platforms_for_version(self):
       ['debian-10-buster', 'ubuntu-19.04-disco'],
     )
     # incompatible platforms are excluded
-    input = {'platforms': ['ubuntu-19.10-eoan', 'ubuntu-14.04-trusty']}
+    input = {'platforms': ['ubuntu-20.04-focal', 'ubuntu-14.04-trusty']}
```

Comment:
Some of these tests need manual updates from time to time, which is not great, but better than not having tests or having tests where too much stuff is mocked out, IMHO.